ui/app/components/ui/tooltip.js

```diff
@@ -1,22 +1,25 @@
-const Component = require('react').Component
-const h = require('react-hyperscript')
-const inherits = require('util').inherits
-const ReactTooltip = require('react-tooltip-component')
+import PropTypes from 'prop-types'
+import React from 'react'
+import ReactTooltip from 'react-tooltip-component'
 
-module.exports = Tooltip
-
-inherits(Tooltip, Component)
-function Tooltip () {
-  Component.call(this)
+function Tooltip ({ position, title, children }) {
+  return (
+    <ReactTooltip position={position} title={title} fixed>
+      {children}
+    </ReactTooltip>
+  )
 }
 
-Tooltip.prototype.render = function () {
-  const props = this.props
-  const { position, title, children } = props
+Tooltip.defaultProps = {
+  position: 'left',
+  title: '',
+  children: null,
+}
 
-  return h(ReactTooltip, {
-    position: position || 'left',
-    title,
-    fixed: true,
-  }, children)
+Tooltip.propTypes = {
+  position: PropTypes.string,
+  title: PropTypes.string,
```

Comment:
Should this be required, instead of having a default of `''`? I'm not sure what good an empty Tooltip would do. From the look of the component, it'll just... never appear if you pass an empty title.

Reply:
We can, I was err-ing on the side of caution so as to not have propType warnings but I can make this required.

Reply:
I did a quick check of everywhere we're using this, and it _looks_ like it's always a non-empty string. Mostly localized messages.